package.json

```diff
@@ -52,6 +52,7 @@
     "lint": "eslint",
     "test": "scripts/test.sh",
     "package": "npm ci --build-from-source=realm && node-pre-gyp package",
+    "prepare": "lerna bootstrap",
```

Comment:
this reflects on the Realm package. I would prefer we don't introduce such changes until we have time to fix our repo with our long term decision. This task was dropped in the Q3 planning.

Reply:
```suggestion
```